server-spi-private/src/main/java/org/keycloak/broker/provider/AbstractIdentityProviderMapper.java

```diff
@@ -65,7 +65,7 @@ private void registerConfigSyncEventListenerOnce(KeycloakSessionFactory factory)
                 if (keycloakSessionFactory != factory) {
                     keycloakSessionFactory = factory;
 
-                    LOG.infof("Registering %s", ConfigSyncEventListener.class);
+                    LOG.debugf("Registering %s", ConfigSyncEventListener.class);
```

Comment:
I would suggest not to do changes like that as it is not too relevant to the Quarkus upgrade, AFAIK.

Reply:
But it's not a blocker of course.

Reply:
> Do you want me to remove all the SuppressWarnings for this PR? Agree it's more a polishing than strictly tied to the Quarkus upgrade.

Nah, let's keep them. They're all mostly Quarkus related, so it's still kinda relevant to the Quarkus upgrade.